/*
 * This file was automatically generated by EvoSuite
 * Tue Apr 09 07:47:52 GMT 2024
 */

package br.com.swconsultoria.certificado.exception;

import org.junit.Test;
import static org.junit.Assert.*;
import br.com.swconsultoria.certificado.exception.CertificadoException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class CertificadoException_ESTest extends CertificadoException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      CertificadoException certificadoException0 = new CertificadoException("l&@W]E>eEd(=");
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      CertificadoException certificadoException0 = new CertificadoException("l&@W]E>eEd(=", (Throwable) null);
      CertificadoException certificadoException1 = new CertificadoException(certificadoException0);
      assertFalse(certificadoException1.equals((Object)certificadoException0));
  }
}
